README.md

```diff
@@ -1,2 +1,274 @@
+3. Remove the smaller cubes on the center of each face, and the smaller cube on the very center of the larger cube (leaving a total of 20 cubes).
+4. Now, repeat steps 2 and 3 for each smaller cube infinite times an you have the fractal!
+
+The first iteration of this loop gives you a level-1 sponge, the second one gives you a level-2 sponge, and so on. The Menger Sponge itself is the resulting shape after infinite iterations.
+
+### Menger Brocolli
+
+![gif](./media/menger_brocolli.gif)
+
+📌 This is not a canonical fractal. It was constructed by me while I played with the algorithm I used to render the Menger Sponge and it is not based on any canonical fractal. I named it Menger Brocolli due to the fact that it uses the same algorithm I used for the Menger Sponge, and it also resembles a coliflower or a brocolli
+<br/>
+Unfortunately, the original algorithm I used to make that animaiton was lost, and the one provided on this repository is a mere aproximation (I couldn't find the original paramenters I used to make it like in the video).
+
+🎨 The construction of this fractal will be explained based on the algorithm used, since there is no canonical definition to its shape. It was constructed using a ray-marched Signed Distance Field (SDF) and several other space-altering functinos provided on the shader.
+<br/>
+There are some great resources to learn about these concepts, and they are all mentioned in the documentation of each individual shader. For all of the three-dimensional fractals, the resources I used were:
+<br/>
+- Inigo Quilez: http://iquilezles.org​
+- Sebastian Lague: https://www.youtube.com/user/Cercopithecan
+- Code Parade: https://www.youtube.com/channel/UCrv269YwJzuZL3dH5PCgxUw
+
+### Menger Mushroom
+
+![gif](./media/menger_mushroom.gif)
+
+📌 Like the Menger Brocolli, I constructed this one myself, also based on the same algortihm I used to render the Menger Sponge. In fact, it isn't even one sigle fractal, but a collection of several fractal shapes merging toghether to form the animation. I named it Menger Mushroom also because I used the algorithm from the Menger Sponge as a base, and, for some reason, it reminded me of mushrooms, so I went with it.
+<br/>
+Unlike the brocolli, the shader on this repository is the original shader I used to record the animation.
+
+🎨 The construction of this fractal will be explained based on the algorithm used, since there is no canonical definition to its shape. It was constructed using a ray-marched Signed Distance Field (SDF) and several other space-altering functinos provided on the shader.
+<br/>
+There are some great resources to learn about these concepts, and they are all mentioned in the documentation of each individual shader. For all of the three-dimensional fractals, the resources I used were:
+<br/>
+- Inigo Quilez: http://iquilezles.org​
+- Sebastian Lague: https://www.youtube.com/user/Cercopithecan
+- Code Parade: https://www.youtube.com/channel/UCrv269YwJzuZL3dH5PCgxUw
```

Comment:
Se for pra colocar a mesma referência várias vezes, melhor colocar no final, não?

Reply:
Eu tentei fazer de um jeito que a pessoa poderia clicar no fractal que ela queria e ver tudo que eu tenho sobre ele sem precisar ele a parada toda, por isso coloquei repetido.